emr/application/views/screeners/viewRequests.php

```diff
@@ -0,0 +1,58 @@
+<!-- page content -->
```

Comment:
this file should be named just `view` not `viewRequests`

Reply:
done

Reply:
Looks good!